contrib/devtools/check-deps.sh

```diff
@@ -194,6 +191,7 @@ cd "$BUILD_DIR"
 # shellcheck disable=SC2046
 make -j"$(nproc)" $(lib_targets)
```

Comment:
I think we are trying to consistently use `cmake --build` over `make `.

Reply:
re: https://github.com/bitcoin/bitcoin/pull/30415#discussion_r1745115109

> I think we are trying to consistently use `cmake --build` over `make `.

Thanks, switched to cmake --build